testsuites/listener/shared/client_sg/test_views.py

```diff
@@ -118,3 +120,51 @@ def test_stale_revision_should_not_be_in_the_index(setup_client_syncgateway_test
 
     client.verify_view_contains_keys(view_response=content_view_rows_2, keys=doc["id"])
     client.verify_view_contains_values(view_response=content_view_rows_2, values=updated_doc["rev"])
+
+
+@pytest.mark.listener
+def test_view_query_performance(setup_client_syncgateway_test):
+    """
+     Run this test when new iOS version arrives to make sure CBL query is not diminishing
```

Comment:
Use @summary annotation to provide details about module in doc string.

Reply:
done